.github/workflows/deno.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Not sure this is needed at all? It is not something that the team can actually maintain going forward.

Reply:
OK,I have removed this.